lib/finalize_security_release.js

```diff
@@ -0,0 +1,68 @@
+import {
+  NEXT_SECURITY_RELEASE_FOLDER,
+  NEXT_SECURITY_RELEASE_REPOSITORY,
+  checkoutOnSecurityReleaseBranch,
+  commitAndPushVulnerabilitiesJSON
+} from './security-release/security-release.js';
+import fs from 'node:fs';
+import path from 'node:path';
+
+export default class FinalizeSecurityRelease {
```

Comment:
```suggestion
export default class UpdateSecurityRelease {
```

we could rename it to `UpdateSecurityRelease`

then the correct `start` function transform into `updateReleaseDate`